plugins/search/src/components/SearchModal/SearchModal.test.tsx

```diff
@@ -55,7 +55,7 @@ describe('SearchModal', () => {
     );
 
     expect(screen.getByRole('dialog')).toBeInTheDocument();
-    expect(query).toHaveBeenCalledTimes(1);
+    expect(query).toHaveBeenCalledTimes(2);
```

Comment:
Was this meant to change? what causes the call to increase from 1 - 2?

Reply:
The local context state was created even when we used the parent, so I extracted it to a separate component, so now it is only called once.